ai-engineering/measure.mdx

```diff
@@ -1,34 +1,45 @@
+The **Measure** stage is where you quantify the quality and effectiveness of your AI <Tooltip tip={definitions.Capability}>capability</Tooltip>. Instead of relying on anecdotal checks, this stage uses a systematic process called an <Tooltip tip={definitions.Eval}>eval</Tooltip> to score your capability's performance against a known set of correct examples (<Tooltip tip={definitions.GroundTruth}>ground truth</Tooltip>). This provides a data-driven benchmark to ensure a capability is ready for production and to track its quality over time.
+
+Evaluations (evals) are systematic tests that measure how well your AI features perform. Instead of manually testing AI outputs, evals automatically run your AI code against test datasets and score the results using custom metrics. This lets you catch regressions, compare different approaches, and confidently improve your AI features over time.
 
 ## The `Eval` function
 
-<Badge>Coming soon</Badge> The primary tool for the Measure stage is the `Eval` function, which will be available in the `axiom/ai` package. It provides a simple, declarative way to define a test suite for your capability directly in your codebase.
+The primary tool for the Measure stage is the `Eval` function, available in the `axiom/ai/evals` package. It provides a simple, declarative way to define a test suite for your capability directly in your codebase.
 
 An `Eval` is structured around a few key parameters:
 
-* `data`: An async function that returns your `collection` of `{ input, expected }` pairs, which serve as your ground truth.
+* `data`: An async function that returns your collection of `{ input, expected }` pairs, which serve as your ground truth.
 * `task`: The function that executes your AI capability, taking an `input` and producing an `output`.
-* `scorers`: An array of `grader` functions that score the `output` against the `expected` value.
-* `threshold`: A score between 0 and 1 that determines the pass/fail condition for the evaluation.
+* `scorers`: An array of scorer functions that score the `output` against the `expected` value.
+* `metadata`: Optional metadata for the evaluation, such as a description.
 
 Here is an example of a complete evaluation suite:
 
 ```ts /evals/text-match.eval.ts
-import { Levenshtein } from 'autoevals';
-import { Eval } from 'axiom/ai/evals';
+import { Eval, Scorer } from 'axiom/ai/evals';
+
+const LevenshteinScorer = Scorer(
+  'Levenshtein',
+  ({ output, expected }: { output: string; expected: string }) => {
+    // Calculate Levenshtein distance score
+    const distance = calculateLevenshtein(output, expected);
+    const maxLen = Math.max(output.length, expected.length);
+    return maxLen === 0 ? 1 : 1 - distance / maxLen;
+  }
+);
 
 Eval('text-match-eval', {
```

Comment:
Should we encourage users to add capability name to metadata?

Reply:
IMO capability should be either
- parsed automatically by traversing the trace
- put somewhere more prominent than metadata, which imo is a place for the user to stick whatever they want